apps/api/src/billing/services/usage/usage.service.spec.ts

```diff
@@ -0,0 +1,282 @@
+import type { UsageRepository } from "@src/billing/repositories/usage/usage.repository";
+import type { LeaseRepository } from "@src/deployment/repositories/lease/lease.repository";
+import { UsageService } from "./usage.service";
+
+import { AkashAddressSeeder } from "@test/seeders/akash-address.seeder";
+
+describe(UsageService.name, () => {
+  let service: UsageService;
+  let usageRepository: jest.Mocked<UsageRepository>;
+  let leaseRepository: jest.Mocked<LeaseRepository>;
+
+  beforeEach(() => {
```

Comment:
Pls use setup function approach, avoid jest hooks. This is smth we follow for all cases.

Reply:
Done